packages/commonwealth/client/scripts/state/api/communities/updateCustomDomain.tsx

```diff
@@ -0,0 +1,7 @@
+import { trpc } from 'utils/trpcClient';
+
+const useUpdateCustomDomainMutation = () => {
+  return trpc.community.updateCustomDomain.useMutation();
```

Comment:
you dont need invalidate any cache after this mutation?

Reply:
I will disable caching on the query with `staleTime: 0` becuase we don't really need this to be cached, and it will only be used by super admins

Reply:
In this case you should set staleTime to Infinity instead of 0. Zero means that your query is invalidated right away so react-query will try to fetch again